portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -64,17 +81,24 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String phone = getParameter(request, "phone", "");
     String message = getParameter(request, "message", "");
 
-    Entity taskEntity = new Entity("Task");
-    taskEntity.setProperty("first-name", text);
-    taskEntity.setProperty("last-name", text2);
-    taskEntity.setProperty("email", email);
-    taskEntity.setProperty("phone", phone);
-    taskEntity.setProperty("message", message);
+    Entity commentEntity = new Entity("Comments");
```

Comment:
same idea - make into conversion function

Reply:
I created a helper function for this as well :) Thanks